source/nodejs/adaptivecards/src/card-elements.ts

```
@@ -1562,10 +1562,10 @@ export class NumberInput extends Input {
 
     protected internalRender(): HTMLElement {
         this._numberInputElement = document.createElement("input");
-        this._numberInputElement.type = "number";
+        this._numberInputElement.setAttribute('type',"number");
```

Quote consistency. Using single and double on the same line
Space after comma

---

BTW is that setAttribute also something incompatible with a particular browser?

---

The problem is setting type to "date" will throw exception in IE causing entire card failed to render but setAttribute will fail silently and keep rendering as a normal input.

---

and setAttribute has better compatibility which is even supported by IE8